### PR TITLE
Pooling padding.Resolves #91

### LINKED ISSUE #91: Padding for pooling layers
Now Conv2D allows explicit padding, while pooling layers are not. We need consistent padding types for both convolution and pooling layers.

### PR DESCRIPTION
**Pull request by david-berthelot**
Resolves #91

### CODE REVIEW

File: tests/functional_pooling.py [hunk 1]
```diff
@@ -16,14 +16,14 @@
 
 import unittest
 
-import jax.numpy as jn
+import jax.numpy as  jn
```
Comment (david-berthelot) [inline tests/functional_pooling.py:19]:
```suggestion
import jax.numpy as jn
```

**Review by kihyuks — APPROVED** [review]
LGTM
